src/main/java/de/taimos/pipeline/aws/S3UploadStep.java

```diff
@@ -74,6 +77,7 @@
 	private String excludePathPattern;
 	private String workingDir;
 	private String[] metadatas;
+	private String tags;
```

Comment:
why not model this as a Map<String, String> ?

Reply:
Was originally following the metadatas design.  Its basically the same discussion there, if you would like, will try a Map<string,string> and let you know

Reply:
Tried looking at changing this, but because this is coming in via the pipeline as a string, we then need to parse the string.

I assume we want the databound methods to continue to work:
```
	public String getTags() {
		if (this.tags != null) {
			return this.tags;
		} else {
			return null;
		}
	}


	@DataBoundSetter
	public void setTags(String tags) {
		if (tags != null ) {
			this.tags = tags;
		} else {
			this.tags = null;
		}
	}
```

This enables the pipeline syntax viewer to work as expected.  So let me know what you think.  Everytime I try to change to map, it breaks the pipeline syntax viewer.